assets/src/application/ui/layout/entityList/EntityTable.tsx

```diff
@@ -14,21 +15,25 @@ const EntityTable = <E extends Entity, FS extends ELFSM>({
 	entities,
 	filterState,
 	headerRowGenerator,
+	onSort,
 	tableCaption,
 	tableId,
 }: EntityTableProps<E, FS>) => {
 	const bodyRows = entities.map((entity) => bodyRowGenerator({ entity, filterState }));
 	const headerRow = headerRowGenerator(filterState);
 
+	const newclassName = classNames(className, 'ee-entity-table');
```

Comment:
why not:
```
	const className = classNames(props.className, 'ee-entity-table');
```
and `className` can be removed from the destructuring and `...props` can be added instead?
***
the reason being is that `newclassName` doesn't help/describe the logic here, it's rather `enhancedClassName`, dunno

Reply:
Done

Reply:
Love when the actual implementation is better than the initial suggestion, well done :+1: